<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layout" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>171</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255)</string>
    </property>
    <widget class="QPushButton" name="searchButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;맑은 고딕&quot;;
background-color:qlineargradient(spread:pad, x0:1, y0:1, x0:1, y1:1, stop:0 rgba(170, 170, 255, 200), stop:1 rgba(255, 255, 255, 200))</string>
     </property>
     <property name="text">
      <string>검색(Search)</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>151</width>
       <height>401</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;맑은 고딕&quot;;</string>
     </property>
     <property name="title">
      <string>검색할 정보</string>
     </property>
     <widget class="QLabel" name="cityLabel">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>64</width>
        <height>15</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;맑은 고딕&quot;;</string>
      </property>
      <property name="text">
       <string>시군구</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cityBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;맑은 고딕&quot;;</string>
      </property>
     </widget>
     <widget class="QLabel" name="dongLabel">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>64</width>
        <height>15</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;맑은 고딕&quot;;</string>
      </property>
      <property name="text">
       <string>법정동</string>
      </property>
     </widget>
     <widget class="QComboBox" name="dongBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;맑은 고딕&quot;;</string>
      </property>
     </widget>
     <widget class="QLabel" name="LCategoryLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;맑은 고딕&quot;;</string>
      </property>
      <property name="text">
       <string>업종대분류</string>
      </property>
     </widget>
     <widget class="QComboBox" name="LCategoryBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;맑은 고딕&quot;;</string>
      </property>
     </widget>
     <widget class="QLabel" name="MCategoryLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;맑은 고딕&quot;;</string>
      </property>
      <property name="text">
       <string>업종중분류</string>
      </property>
     </widget>
     <widget class="QComboBox" name="MCategoryBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;맑은 고딕&quot;;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="SCategoryBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;맑은 고딕&quot;;</string>
      </property>
     </widget>
     <widget class="QLabel" name="SCategoryLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;맑은 고딕&quot;;</string>
      </property>
      <property name="text">
       <string>업종소분류</string>
      </property>
     </widget>
     <widget class="QPushButton" name="plusButton">
      <property name="geometry">
       <rect>
        <x>92</x>
        <y>330</y>
        <width>31</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="minusButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>330</y>
        <width>31</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="backButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>470</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;맑은 고딕&quot;;
background-color:qlineargradient(spread:pad, x0:1, y0:1, x0:1, y1:1, stop:0 rgba(170, 170, 255, 200), stop:1 rgba(255, 255, 255, 200))</string>
     </property>
     <property name="text">
      <string>뒤로가기</string>
     </property>
    </widget>
   </widget>
   <widget class="QWebEngineView" name="webView">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>741</width>
      <height>521</height>
     </rect>
    </property>
    <property name="url">
     <url>
      <string>about:blank</string>
     </url>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>912</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
